__tests__/server.test.js

```diff
@@ -0,0 +1,46 @@
+const request = require('supertest')
+const app = require('../server');
+const db = require('../db/connection');
+const data = require('../db/data/test-data/index');
+const seed = require('../db/seeds/seed');
+
+beforeAll(() => {
+    return seed(data).then();
```

Comment:
.then() not needed here

Reply:
Also beforeEach we should be seeding rather than beforeAll

Reply:
Updated both.